Examples/HelloWorld/CMakeLists.txt

```diff
@@ -6,3 +6,45 @@ target_link_libraries ( HelloWorld ${SimpleITK_LIBRARIES} )
 if ( WRAP_CSHARP )
   csharp_add_executable( HelloWorldCS SimpleITKCSharpManaged.dll ${CMAKE_CURRENT_SOURCE_DIR}/HelloWorld.cs )
 endif ()
+
+if ( NOT BUILD_TESTING )
+  return ()
+endif ()
+
+if ( SimpleITK_SHOW_TESTING_ENABLED )
+
+  sitk_add_test( NAME CXX.Example.HelloWorld
+    COMMAND $<TARGET_FILE:HelloWorld>
+    )
+
+  if ( WRAP_CSHARP )
```

Comment:
This `WRAP_CSHARP` should not be needed the `sitk_add_csharp_test` macro should take care of it.